tests/unit_tests/db_engine_specs/test_couchbase.py

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+from typing import Optional
+
+import pytest
+
+from tests.unit_tests.db_engine_specs.utils import assert_convert_dttm
+from tests.unit_tests.fixtures.common import dttm  # noqa: F401
+
+
+def test_epoch_to_dttm() -> None:
+    """
+    DB Eng Specs (couchbase): Test epoch to dttm
+    """
+    from superset.db_engine_specs.couchbasedb import CouchbaseDbEngineSpec
+
+    assert CouchbaseDbEngineSpec.epoch_to_dttm() == "MILLIS_TO_STR({col} * 1000, '111')"
+
+
+def test_epoch_ms_to_dttm() -> None:
+    """
+    DB Eng Specs (couchbase): Test epoch ms to dttm
+    """
+    from superset.db_engine_specs.couchbasedb import CouchbaseDbEngineSpec
+
+    assert CouchbaseDbEngineSpec.epoch_ms_to_dttm() == "MILLIS_TO_STR({col}, '111')"
+
+
+@pytest.mark.parametrize(
+    "target_type,expected_result",
+    [
+        ("Date", "DATETIME(DATE_FORMAT_STR(STR_TO_UTC('2019-01-02'), 'iso8601'))"),
+        (
+            "DateTime",
+            "DATETIME(DATE_FORMAT_STR(STR_TO_UTC('2019-01-02T03:04:05'), 'iso8601'))",
+        ),
```

Comment:
It would be nice to have a few more typical formats featured here, like `VARCHAR`, `TEXT`, `INT`, as I assume those will be commonly used.

Reply:
I have added test case for other data types also.